nautobot/extras/forms/forms.py

```diff
@@ -605,10 +609,17 @@ class ExternalIntegrationBulkEditForm(NautobotBulkEditForm):
     verify_ssl = forms.NullBooleanField(required=False, label="Verify SSL", widget=BulkEditNullBooleanSelect)
     timeout = forms.IntegerField(required=False, min_value=0)
     extra_config = forms.JSONField(required=False)
+    http_method = forms.ChoiceField(
+        required=False,
+        label="HTTP Method",
+        choices=add_blank_choice(HTTPMethodClassChoices),
+    )
+    headers = forms.JSONField(required=False, label="HTTP Request headers")
+    ca_file_path = forms.CharField(required=False, label="CA File Path")
```

Comment:
Is this order of fields in the form the best/most logical way to structure them? I would have thought `extra_config` at least should be the last in the form.